Cannot invoke "net.minecraft.class_1922.method_8320(net.minecraft.class_2338)" because "world" is null

## Some information
Operating system: Windows 10
Java version: 17
Minecraft version: 1.19.2
Baritone version: 1.9.4
Other mods (if used): [Prominence Fabric Modpack](https://www.curseforge.com/minecraft/modpacks/prominence-fabric)

## Exception, error or logs
Uncaught exception in thread "pool-9-thread-3"
java.lang.NullPointerException: Cannot invoke "net.minecraft.class_1922.method_8320(net.minecraft.class_2338)" because "world" is null
	at net.minecraft.class_4970$class_4971.handler$hlc000$interference$getHardness(class_4970.java:5653) ~[client-intermediary.jar:?]
	at net.minecraft.class_4970$class_4971.method_26214(class_4970.java) ~[client-intermediary.jar:?]
	at baritone.jp.a(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at baritone.jp.a(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at baritone.jp.a(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at java.util.function.Function.lambda$compose$0(Unknown Source) ~[?:?]
	at java.util.HashMap.computeIfAbsent(Unknown Source) ~[?:?]
	at baritone.jp.a(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at baritone.gp.a(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at baritone.iv.a(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at baritone.iv.d(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) ~[?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) ~[?:?]
	at java.util.stream.DistinctOps$1$2.accept(Unknown Source) ~[?:?]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(Unknown Source) ~[?:?]
	at baritone.iv.a(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at baritone.iv.a(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at baritone.iv.a(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at baritone.iv.b(Unknown Source) ~[baritone-standalone-fabric-1.9.4.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[?:?]
	at java.lang.Thread.run(Unknown Source) [?:?]

## How to reproduce
Use modpack
Try to #mine 

## Modified settings
None

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have included the version of Minecraft I'm running, baritone's version and forge mods (if used).
- [x] I have included logs, exceptions and / or steps to reproduce the issue.
- [x] I have not used any OwO's or UwU's in this issue.

[latest.log](https://github.com/cabaletta/baritone/files/11447515/latest.log)

`[21:21:11] [Render thread/INFO]: [CHAT] [Baritone] > mine diamond_ore
[21:21:11] [Render thread/INFO]: [CHAT] [Baritone] > mine diamond_ore
[21:21:11] [Render thread/INFO]: ARRP register - before vanilla
[21:21:11] [Render thread/INFO]: ARRP register - after vanilla
[21:21:11] [Render thread/INFO]: Reloading ResourceManager: Supplementaries Generated Pack
[21:21:11] [Render thread/INFO]: closing rrp minecraft:extrasounds
[21:21:11] [Render thread/INFO]: closing rrp woof:arrp
[21:21:11] [Render thread/INFO]: ARRP register - before vanilla
[21:21:11] [Render thread/INFO]: ARRP register - after vanilla
[21:21:11] [Render thread/WARN]: Could not resolve dependencies for listener: maybedata:conditional_recipes!
[21:21:11] [Render thread/WARN]: Could not resolve dependencies for listener: trinkets:entities!
[21:21:11] [Thread-181/INFO]: Loaded 0 item attribute modifiers
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/black_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/brown_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/cyan_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/yellow_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/green_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/lime_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/orange_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/white_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/light_blue_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/light_gray_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/gray_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/pink_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/magenta_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/purple_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/blue_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-185/WARN]: Ignoring loot modifiers for:woof:tables/block/red_dog_bed use this folder name:lootintegrations
[21:21:11] [Thread-193/ERROR]: [Fabric Seasons] Failed to load hardcoded grass colors
java.util.NoSuchElementException: No value present
	at java.util.Optional.orElseThrow(Unknown Source) ~[?:?]
	at io.github.lucaargolo.seasons.resources.CropConfigs.method_14491(CropConfigs.java:69) ~[fabric-seasons-2.1+1.19.jar:?]
	at net.minecraft.class_4013.method_29490(class_4013.java:15) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(Unknown Source) ~[?:?]
	at java.util.concurrent.CompletableFuture$Completion.run(Unknown Source) ~[?:?]
	at net.minecraft.class_4014.method_18365(class_4014.java:69) ~[client-intermediary.jar:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
[21:21:11] [ForkJoinPool.commonPool-worker-9/INFO]: text.translated_server.loaded.translation_key
[21:21:11] [Thread-204/INFO]: Loaded 0 survival effects
[21:21:11] [Thread-206/INFO]: Loaded 0 trophies
[21:21:11] [Thread-207/INFO]: Reloaded configs for wandering_collector (server/smallest)
[21:21:11] [Thread-208/INFO]: Loaded 0 Stalactite configs!
[21:21:11] [Render thread/INFO]: [CHAT] [Baritone] Mining [BlockOptionalMeta{block=Block{minecraft:diamond_ore}}]
[21:21:11] [Render thread/INFO]: [CHAT] [Baritone] Mining [BlockOptionalMeta{block=Block{minecraft:diamond_ore}}]
[21:21:12] [Render thread/INFO]: [CHAT] [Baritone] An unhandled exception occurred. The error is in your game's log, please report this at https://github.com/cabaletta/baritone/issues
[21:21:12] [Render thread/INFO]: [CHAT] [Baritone] An unhandled exception occurred. The error is in your game's log, please report this at https://github.com/cabaletta/baritone/issues
[21:21:12] [Render thread/INFO]: [STDERR]: baritone.api.command.exception.CommandUnhandledException: java.lang.NullPointerException: Cannot invoke "net.minecraft.class_1922.method_8320(net.minecraft.class_2338)" because "world" is null
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.command.manager.CommandManager$ExecutionWrapper.execute(CommandManager.java:147)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.command.manager.CommandManager.execute(CommandManager.java:84)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.command.ExampleBaritoneControl.runCommand(ExampleBaritoneControl.java:142)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.command.ExampleBaritoneControl.onSendChatMessage(ExampleBaritoneControl.java:67)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.event.GameEventHandler.lambda$onSendChatMessage$2(GameEventHandler.java:69)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.concurrent.CopyOnWriteArrayList.forEach(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.event.GameEventHandler.onSendChatMessage(GameEventHandler.java:69)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_746.handler$bjn000$baritone$sendChatMessage(class_746.java:4922)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_746.method_44097(class_746.java)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_746.method_44096(class_746.java:324)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_408.method_44056(class_408.java:519)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_408.method_25404(class_408.java:213)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_309.method_1454(class_309.java:374)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_437.method_25412(class_437.java:489)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_309.method_1466(class_309.java:370)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_309.method_22678(class_309.java:462)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_1255.execute(class_1255.java:103)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_309.method_22676(class_309.java:462)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:44)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at org.lwjgl.system.JNI.invokeV(Native Method)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3403)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:197)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_1041.method_15998(class_1041.java:315)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1523(class_310.java:1205)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1514(class_310.java:768)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.method_44604(Main.java:244)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(Main.java:51)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
[21:21:12] [Render thread/INFO]: [STDERR]: Caused by: java.lang.NullPointerException: Cannot invoke "net.minecraft.class_1922.method_8320(net.minecraft.class_2338)" because "world" is null
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_4970$class_4971.handler$hlb000$interference$getHardness(class_4970.java:5653)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_4970$class_4971.method_26214(class_4970.java)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.utils.ToolSet.calculateSpeedVsBlock(ToolSet.java:180)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.utils.ToolSet.getBestSlot(ToolSet.java:135)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.utils.ToolSet.getBestDestructionTime(ToolSet.java:163)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.function.Function.lambda$compose$0(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.HashMap.computeIfAbsent(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.utils.ToolSet.getStrVsBlock(ToolSet.java:75)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.pathing.movement.MovementHelper.getMiningDurationTicks(MovementHelper.java:606)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.process.MineProcess.plausibleToBreak(MineProcess.java:469)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.process.MineProcess.lambda$prune$11(MineProcess.java:429)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.stream.ReferencePipeline$2$1.accept(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.stream.DistinctOps$1$2.accept(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.process.MineProcess.prune(MineProcess.java:444)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.process.MineProcess.searchWorld(MineProcess.java:377)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.process.MineProcess.rescan(MineProcess.java:238)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.process.MineProcess.mine(MineProcess.java:495)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.api.process.IMineProcess.mine(IMineProcess.java:75)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.command.defaults.MineCommand.execute(MineCommand.java:50)
[21:21:12] [Render thread/INFO]: [STDERR]: 	at baritone.command.manager.CommandManager$ExecutionWrapper.execute(CommandManager.java:142)
[21:21:12] [Render thread/INFO]: [STDERR]: 	... 29 more
[21:21:12] [pool-9-thread-4/INFO]: [STDOUT]: 22 movements considered
[21:21:12] [pool-9-thread-4/INFO]: [STDOUT]: Open set size: 0
[21:21:12] [pool-9-thread-4/INFO]: [STDOUT]: PathNode map size: 1
[21:21:12] [pool-9-thread-4/INFO]: [STDOUT]: 52 nodes per second
[21:21:12] [Render thread/INFO]: [CHAT] [Baritone] Unable to find any path to BlockOptionalMetaLookup{[BlockOptionalMeta{block=Block{minecraft:diamond_ore}}]}, canceling mine
[21:21:12] [Render thread/INFO]: [CHAT] [Baritone] Unable to find any path to BlockOptionalMetaLookup{[BlockOptionalMeta{block=Block{minecraft:diamond_ore}}]}, canceling mine
[21:21:12] [pool-9-thread-3/ERROR]: Uncaught exception in thread "pool-9-thread-3"
java.lang.NullPointerException: Cannot invoke "net.minecraft.class_1922.method_8320(net.minecraft.class_2338)" because "world" is null
	at net.minecraft.class_4970$class_4971.handler$hlb000$interference$getHardness(class_4970.java:5653) ~[client-intermediary.jar:?]
	at net.minecraft.class_4970$class_4971.method_26214(class_4970.java) ~[client-intermediary.jar:?]
	at baritone.utils.ToolSet.calculateSpeedVsBlock(ToolSet.java:180) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.utils.ToolSet.getBestSlot(ToolSet.java:135) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.utils.ToolSet.getBestDestructionTime(ToolSet.java:163) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at java.util.function.Function.lambda$compose$0(Unknown Source) ~[?:?]
	at java.util.HashMap.computeIfAbsent(Unknown Source) ~[?:?]
	at baritone.utils.ToolSet.getStrVsBlock(ToolSet.java:75) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.pathing.movement.MovementHelper.getMiningDurationTicks(MovementHelper.java:606) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.process.MineProcess.plausibleToBreak(MineProcess.java:469) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.process.MineProcess.lambda$prune$11(MineProcess.java:429) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) ~[?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) ~[?:?]
	at java.util.stream.DistinctOps$1$2.accept(Unknown Source) ~[?:?]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(Unknown Source) ~[?:?]
	at baritone.process.MineProcess.prune(MineProcess.java:444) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.process.MineProcess.searchWorld(MineProcess.java:377) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.process.MineProcess.rescan(MineProcess.java:238) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.process.MineProcess.lambda$onTick$1(MineProcess.java:112) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[?:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
[21:21:12] [pool-9-thread-5/ERROR]: Uncaught exception in thread "pool-9-thread-5"
java.lang.NullPointerException: Cannot invoke "net.minecraft.class_1922.method_8320(net.minecraft.class_2338)" because "world" is null
	at net.minecraft.class_4970$class_4971.handler$hlb000$interference$getHardness(class_4970.java:5653) ~[client-intermediary.jar:?]
	at net.minecraft.class_4970$class_4971.method_26214(class_4970.java) ~[client-intermediary.jar:?]
	at baritone.utils.ToolSet.calculateSpeedVsBlock(ToolSet.java:180) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.utils.ToolSet.getBestSlot(ToolSet.java:135) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.utils.ToolSet.getBestDestructionTime(ToolSet.java:163) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at java.util.function.Function.lambda$compose$0(Unknown Source) ~[?:?]
	at java.util.HashMap.computeIfAbsent(Unknown Source) ~[?:?]
	at baritone.utils.ToolSet.getStrVsBlock(ToolSet.java:75) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.pathing.movement.MovementHelper.getMiningDurationTicks(MovementHelper.java:606) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.process.MineProcess.plausibleToBreak(MineProcess.java:469) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.process.MineProcess.lambda$prune$11(MineProcess.java:429) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) ~[?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(Unknown Source) ~[?:?]
	at java.util.stream.DistinctOps$1$2.accept(Unknown Source) ~[?:?]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(Unknown Source) ~[?:?]
	at baritone.process.MineProcess.prune(MineProcess.java:444) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.process.MineProcess.searchWorld(MineProcess.java:377) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.process.MineProcess.rescan(MineProcess.java:238) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at baritone.process.MineProcess.lambda$onTick$1(MineProcess.java:112) ~[baritone-unoptimized-fabric-1.9.4.jar:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[?:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
[21:21:14] [Render thread/INFO]: [CHAT] [Baritone] > stop
[21:21:14] [Render thread/INFO]: [CHAT] [Baritone] > stop
[21:21:14] [Render thread/INFO]: [CHAT] [Baritone] ok canceled
[21:21:14] [Render thread/INFO]: [CHAT] [Baritone] ok canceled`

I'm still getting it.

--- a/src/main/java/baritone/utils/ToolSet.java
+++ b/src/main/java/baritone/utils/ToolSet.java
@@ -177,7 +177,13 @@ public class ToolSet {
      * @return how long it would take in ticks
      */
     public static double calculateSpeedVsBlock(ItemStack item, BlockState state) {
-        float hardness = state.getDestroySpeed(null, null);
+        float hardness;
+        try {
+            hardness = state.getDestroySpeed(null, null);
+        } catch (NullPointerException npe) {
+            // can't easily determine the hardness so treat it as unbreakable
+            return -1;
+        }
         if (hardness < 0) {
             return -1;
         }

